### PR TITLE
PrimVtx and ITSTPC matching json to run in multinode; includes also json to be added to consul, to run at P2

### PR DESCRIPTION
**Pull request by chiarazampolli**
@knopers8 : can you check? This is what we discussed. 
Local tests seem to indicate a race condition: the remote node cannot receive the data, it only sees End Of Stream. But since there is no other way to test it, maybe it is ok to commit it?
I include here also [GLO_mnl_EPN.json](https://github.com/AliceO2Group/QualityControl/compare/master...chiarazampolli:json_P2?expand=1#diff-a49b23af8eea2badf5f942a67b0db906bd92c59102a4d369ac99f1b2ef78535e), which is what I will put in consul, if you agree. 

To test:

PrimVertex:
local node:
```
o2-primary-vertex-reader-workflow --hbfutils-config o2_tfidinfo.root -b | o2-qc --config json:///home/zampolli/SOFT/alibuild/ali-o2-dev1/QualityControl/Modules/GLO/vertexing-qc_mnl.json -b --local --host localhost
```

remote node, from another terminal:
```
o2-qc --config json:///home/zampolli/SOFT/alibuild/ali-o2-dev1/QualityControl/Modules/GLO/vertexing-qc_mnl.json -b --remote
```

Matching:
local node:
```
o2-global-track-cluster-reader --track-types "ITS-TPC,ITS,TPC" --cluster-types none -b --hbfutils-config o2_tfidinfo.root | o2-qc --config json:////home/zampolli/SOFT/alibuild/ali-o2-dev1/QualityControl/Modules/GLO/ITSTPCmatchedTracks_mnl.json -b --local --host localhost
```

remote node, from another terminal:
o2-qc --config json:////home/zampolli/SOFT/alibuild/ali-o2-dev1/QualityControl/Modules/GLO/ITSTPCmatchedTracks_mnl.json -b --remote
```

(of course, you need the prim vtx, and tracks files).

### CODE REVIEW

File: Modules/GLO/GLO_mnl_EPN.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	"taskParameters" : {
+          "isMC": "false"
+        },
+        "location" : "local",
+	"localMachines": [
+            "epn",
+	    "localhost"
+        ],
+        "remoteMachine": "alio2-cr1-qc03.cern.ch",
+	"remotePort": "46000",
```
Comment (knopers8) [inline Modules/GLO/GLO_mnl_EPN.json:51]: please use a port in range of 47700-47799 which is not yet used by a different task. You can grep ControlWorkflows/scripts/etc for those.

This also applies to the same fields in other tasks/config files.

File: Modules/GLO/GLO_mnl_EPN.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        },
+	"taskParameters" : {
+          "isMC": "false"
+        },
+        "location" : "local",
+	"localMachines": [
+            "epn",
+	    "localhost"
+        ],
+        "remoteMachine": "alio2-cr1-qc03.cern.ch",
```
Comment (knopers8) [inline Modules/GLO/GLO_mnl_EPN.json:50]: I will assign you `alio2-cr1-qc07.cern.ch`.

File: Modules/GLO/GLO_mnl_EPN.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      "taskParameters" : {
+        "GID" : "ITS-TPC,ITS",
+        "verbose" : "false",
+        "minPtCut" : "0.1f",
+        "etaCut" : "1.4f",
+        "minNTPCClustersCut" : "60",
+        "minDCACut" : "100.f",
+        "minDCACutY" : "10.f",
+        "grpFileName" : "o2sim_grp.root",
+        "geomFileName" : "o2sim_geometry-aligned.root"
```
Comment (knopers8) [inline Modules/GLO/GLO_mnl_EPN.json:75]: will these files be accessible on QC nodes?

File: Modules/GLO/ITSTPCmatchedTracks_mnl.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      },
+      "conditionDB" : {
+        "url" : "ccdb-test.cern.ch:8080"
+      },
+      "infologger" : { "" : "Configuration of the Infologger (optional).",
+                       "filterDiscardDebug" : "false",
+                       "" : "Set to true to discard debug and trace messages (default: false)",
+                       "filterDiscardLevel" : "21",
+                       "" : "Message at this level or above are discarded (default: 21 - Trace)" }
+    },
```
Comment (knopers8) [inline Modules/GLO/ITSTPCmatchedTracks_mnl.json:29]: if you want to use also this fie at P2, please use the same qc.config section as in `GLO_mnl_EPN.json`

File: Modules/GLO/GLO_mnl_EPN.json [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+        "url" : "influxdb-unix:///tmp/telegraf.sock"
+      },
+      "consul" : {
+        "url" : "http://ali-consul.cern.ch:8500"
+      },
+      "conditionDB" : {
+        "url" : "http://localhost:8084"
+      },
+      "infologger" : { "" : "Configuration of the Infologger (optional).",
+                       "filterDiscardDebug" : "1",
```
Comment (knopers8) [inline Modules/GLO/GLO_mnl_EPN.json:25]: do you have a lot of debug logs? if not, you could keep it at `false`.

**Review by knopers8 — CHANGES_REQUESTED** [review]

File: Modules/GLO/GLO_mnl_EPN.json (hunk 1 above)
Comment (chiarazampolli) [reply Modules/GLO/GLO_mnl_EPN.json:51]: Done.

File: Modules/GLO/GLO_mnl_EPN.json (hunk 2 above)
Comment (chiarazampolli) [reply Modules/GLO/GLO_mnl_EPN.json:50]: Thanks.

File: Modules/GLO/ITSTPCmatchedTracks_mnl.json (hunk 4 above)
Comment (chiarazampolli) [reply Modules/GLO/ITSTPCmatchedTracks_mnl.json:29]: No, this and `vertexing-qc_mnl.json` are for local testing only.

File: Modules/GLO/GLO_mnl_EPN.json (hunk 5 above)
Comment (chiarazampolli) [reply Modules/GLO/GLO_mnl_EPN.json:25]: Yes, it would print too much if I used "false".

File: Modules/GLO/GLO_mnl_EPN.json (hunk 3 above)
Comment (chiarazampolli) [reply Modules/GLO/GLO_mnl_EPN.json:75]: I don't know, but we need those. Also TOF should use them, and maybe someone else. How is it done?
This is related also to https://alice.its.cern.ch/jira/browse/QC-810

Comment (chiarazampolli) [reply Modules/GLO/GLO_mnl_EPN.json]: BTW, these files are needed for processing, no merging, so it does not matter if they are not on the merger node.

Comment (knopers8) [reply Modules/GLO/GLO_mnl_EPN.json]: indeed, correct

**Review by knopers8 — APPROVED** [review]

**Pull request merged**